__tests__/app.test.js

```diff
@@ -107,6 +107,35 @@ describe("app", () => {
 				});
 		});
 	});
+	describe("GET Request - /api/articles/:article_id/comments", () => {
+		test("Status: 200, responds with array of comments for given article_id", () => {
+			return request(app)
+				.get("/api/articles/3/comments")
+				.expect(200)
+				.then(({ body: { comments } }) => {
+					expect(comments).toHaveLength(2);
+					comments.forEach((comment) => {
+						expect(comment).toEqual(
+							expect.objectContaining({
+								comment_id: expect.any(Number),
+								votes: expect.any(Number),
+								created_at: expect.any(String),
+								author: expect.any(String),
+								body: expect.any(String),
+							})
+						);
+					});
+				});
+		});
+		test("Status: 404, responds with error if no comments found", () => {
```

Comment:
I don't want this! not an error. Give me a alllll the comments for article 2. SURE here ya go `{comments:[]}`

but...

give me all the comments for article 99999999. Sorry mate article not found 404